modules/tinymce/src/plugins/accordion/main/ts/core/Actions.ts

```diff
@@ -0,0 +1,38 @@
+import Editor from 'tinymce/core/api/Editor';
+
+import { fireInsertAccordionEvent } from '../api/Events';
+
+const validAncestorContainers = [ 'P', 'H1', 'H2', 'H3', 'H4', 'H5', 'H6', 'UL', 'OL', 'TABLE', 'FIGURE', 'DETAILS', 'DIV', 'DL', 'BODY' ];
+const isValidAncestorContainer = (node: Node): node is HTMLElement => 
+  validAncestorContainers.includes(node.nodeName);
+
+const insertAccordion = (editor: Editor): void => {
+  const dom = editor.dom;
+
+  const rng = editor.selection.getRng();
+  const summaryText = rng.toString() || 'Accordion summary...';
```

Comment:
I don't think we include english strings in content, even the placeholder has to be configured. I'm not sure how we achieve the feature goals of accordions without these strings, though.

Please at least make sure they can be translated.